LLMonFHIR/FHIR Display/MultipleResourceChat.swift

```diff
@@ -0,0 +1,70 @@
+//
+// This source file is part of the Stanford LLM on FHIR project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziOpenAI
+import SwiftUI
+
+
+struct MultipleResourceChat: View {
+    @Environment(\.dismiss) var dismiss
+    @EnvironmentObject var openAPIComponent: OpenAIComponent<FHIR>
+    @EnvironmentObject var fhirResourceSummary: FHIRResourceSummary<FHIR>
+    @EnvironmentObject var fhirStandard: FHIR
+
+    @State var chat: [Chat]
+    @State var gettingAnswer = false
+
+    var body: some View {
+        NavigationStack {
+            ChatView($chat, disableInput: $gettingAnswer)
+                
+                .navigationTitle("All FHIR Resources")
```

Comment:
Please use localized strings using the `Localizable.strings` file when you add new strings to the application.